docs/AdvancedUsage.md

```diff
@@ -0,0 +1,172 @@
+| **ttStartTool**       | **toolName**: String - Select a preconfigured ecu.test or trace.check installation <br/><br/> **workspaceDir**: String - ecu.test or trace.check workspace, relative to build workspace or absolute path.<br/><br/> **settingsDir**: String - ecu.test or trace.check settings directory, relative to build workspace or absolute path.<br/><br/> timeout: int - Maximum time in seconds starting and connecting to the selected tool.<br/><br/> **keepInstance**: boolean - Re-uses an already running ecu.test or trace.check instance with the currently loaded workspace instead of starting a new one.<br/><br/> stopUndefinedTools: boolean - It only has an impact if <i>Keep Previous Instance</i> is unselected.</b> Additionally, all tracetronic tools that are not defined by the Jenkins ETInstallations are stopped. | /                                            |
+| **ttStopTool**        | **toolName**: String - Select a preconfigured ecu.test or trace.check installation <br/><br/> **timeout**: int - Maximum time in seconds terminating the selected tool.<br/><br/> **stopUndefinedTools**: boolean - Additionally, all tracetronic tools that are not defined by the Jenkins ETInstallations are stopped.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | /                                            |
+| **ttUploadReports**   | **testGuideUrl**: String - The URL to the test.guide instance to connect to <br/><br/> **credentialsId**: String - Credentials for test.guide REST API.<br/><br/>**projectId**: int - The test.guide project ID to upload to.<br/><br/> useSettingsFromServer: boolean - Get and use upload settings from test.guide. <br/><br/> additionalSettings:List\<[AdditionalSetting](#additionalsetting)> - Additional ATX generator settings. <br/><br/> **reportIds**: List<String> - List of reportIds to upload for, ignore to upload all.                                                                                                                                                                                                                                                                                            | List\<[UploadResult](#uploadresult)>         |
```

Comment:
`**reportIds**: List<String> - List  ...` you already typed it as a list

Reply:
lets be a bit more specific here. 
`reportdIds: List<String>` was fine. I meant that the description starts with _List of reportIds_. That seems to be the same thing in this context.